declarations.d.ts

```diff
@@ -246,3 +246,30 @@ interface IDynamicHelpProvider {
 interface IMicroTemplateService {
 	parseContent(data: string): string;
 }
+
+interface ISysInfoData {
+
+	procInfo: string; // name and version of the CLI
+
+	// os stuff
+	platform: string; // os.platform
+	os: string; // full os name, like `uname -a` on unix, registry query on win
```

Comment:
Just a suggestion, not a show stopper:
Use javascript api documentation comments
```JavaScript
/** Full os name, like `uname -a` on unix, registry query on win. */
os:string;
```
This way VisualStudio, WebStorm, etc. will show the message when you try using this property.